src/AspNet.Security.OAuth.EVEOnline/EVEOnlineAuthenticationDefaults.cs

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.AspNetCore.Builder;
+
+namespace AspNet.Security.OAuth.EVEOnline {
```

Comment:
I decided to use EVEOnline, just because I contributed this provider previously to OwinOAuthProviders with same casting, but if you wish I change name to camel case.